worker_test.go

```diff
@@ -337,3 +408,13 @@ func waitForServer(port string) {
 	}
 	log.Fatalf("Server on port %s not up after 10 retries", port)
 }
+
+func build(args ...string) (string, error) {
```

Comment:
This function only really needs the following arguments:

- project
- group
- job parameters

The others could be extracted to constants.

Reply:
Indeed, we can extract `host`, `port` to constants and also utilize the global variables `target` and `username`.

So, in the end this becomes `func build(project, group, params string) (string, error) {`